corehq/apps/users/management/commands/add_multi_location_property.py

```diff
@@ -24,22 +24,39 @@ def migrate_user(self, doc):
 
     def migrate_cc_user(self, doc):
 
-        if is_already_migrated(doc):
+        # skip if doesn't have location
+        if not doc['location_id']:
             return
 
+        # skip if already migrated
```

Comment:
I'm not sure this'd be a perfect fit for what you're doing here, but just FYI, if a document is unchanged after being sent through the migration function, `iter_update` won't save it.  So it might be doing essentially this same logic there anyways.
https://github.com/dimagi/commcare-hq/blob/d8262b2cc0ed5c876566f87e21c2fa5139fa129d/corehq/util/couch.py#L301-L301

If a non-trivial amount of time is spent in the `migrate_user` call, it's certainly better to skip here.